pkg/meta/model/bdr.go

```diff
@@ -101,6 +101,7 @@ var BDRActionMap = map[DDLBDRType][]ActionType{
 		ActionReorganizePartition,
 		ActionAlterTablePartitioning,
 		ActionRemovePartitioning,
+		ActionAddVectorIndex,
```

Comment:
Could you please ask TiCDC developers to confirm this?

Reply:
@asddongmen PTAL

Reply:
After being classified as unsafe DDL, TiCDC won't synchronize this DDL in BDR mode. So, it won't affect the original BDR functionality.